AUTHORS.rst

```diff
@@ -1,4 +1,4 @@
 Contributors
 ------------
-
 * Ilya Baryshev <coagulant@mybook.ru>
+* Kazakov Kirill <kirill.kazakov@qoollo.com>
```

Comment:
Could also be changed to markdown, since `README.rst` was.

Reply:
both done